poppy/provider/akamai/certificates.py

```diff
@@ -565,6 +565,20 @@ def delete_certificate(self, cert_obj):
                 )
 
                 if found is False:
+                    if (cert_obj.cert_details["Akamai"]
```

Comment:
Please add a comment what we are doing here. example(check if there is no pending changes in the enrollment